lib/rspec/rails/configuration.rb

```diff
@@ -157,6 +163,18 @@ def filter_rails_from_backtrace!
           filter_gems_from_backtrace "activemodel", "activerecord",
                                      "activesupport", "activejob"
         end
+
+        # @deprecated TestFixtures#fixture_path is deprecated and will be removed in Rails 7.2
+        if ::Rails::VERSION::STRING >= "7.1.0"
+          def config.fixture_path=(path)
```

Comment:
```suggestion
          def fixture_path=(path)
```
We're inside `config.instance_exec`, should this be sufficient?

Reply:
You're right